### PR TITLE
dsv: update role to be more consistent across all streams

### PR DESCRIPTION
**Pull request by miabbott**
Almost all of the streams are using `overlay2` as the docker storage
driver now, so we can remove a lot of the conditional workarounds we
had.

If we can get CAHC to switch to `overlay2` (see
CentOS/sig-atomic-buildscripts#341), we can simplify this role even
more.

### CODE REVIEW

**Comment by miabbott** [comment]
LOL...the `centos` checks appear to be failing due to the `firewalld_service_verify` role, which is fixed in #419 

`fedora/27/atomic` is failing due to a [known docker issue](https://bugzilla.redhat.com/show_bug.cgi?id=1595300)

Some chicken and egg stuff here

File: roles/docker_storage_verify/tasks/main.yml [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
+- name: Set storage driver
+  set_fact:
+    storage_driver: 'overlay2'
 
-- name: Fail if required variables are not defined
-  when: driver is undefined
-  fail:
-    msg: "The required variables are not defined.  Check files under roles/docker_storage_verify/vars"
+- when: ansible_distribution == 'CentOSDev'
+  name: Set storage driver for CAHC
```
Comment (mike-nguyen) [inline roles/docker_storage_verify/tasks/main.yml]: I know we are trying to put the when statement first stylistically, but i think it makes more sense for the name to go first if there is a name, then the actual code underneath.

**Comment by mike-nguyen** [comment]
These are both fairly low risk.  I am okay with merging them back to back!

File: roles/docker_storage_verify/tasks/main.yml (hunk 1 above)
Comment (miabbott) [reply roles/docker_storage_verify/tasks/main.yml]: no big deal, i can make that change

**Pull request merged**